pan3d/dataset_viewer.py

```diff
@@ -67,13 +66,12 @@ def __init__(
         if state:
             self.state.update(state)
 
-        self._pangeo = pangeo
-        self._esgf = esgf
-        if pangeo:
+        print(catalogs)
```

Comment:
remove print

Reply:
Thanks for catching that, this was my quick test of the new kwarg. Removed in [9c893a1](https://github.com/Kitware/pan3d/pull/67/commits/9c893a1abd4812320770177ac71c7d98ab45da5d)